data/persons.txt

```diff
@@ -443,7 +443,7 @@ ship "Lampyrid-Class Transport"
 		"shields" 5400
 		"hull" 4400
 		"required crew" 5
-		"bunks" 14
+		"bunks" 22
```

Comment:
From the linked page, 5 crew + 18 passengers = 23 bunks, not 22.

Reply:
I can do math, I promise! Thanks, fixed.